gnomeExtensions.freon: init at 40

justinas opened the pull request:
##### Motivation for this change
[freon](https://github.com/UshakovVasilii/gnome-shell-extension-freon) is a more up-to-date fork of the [Sensors](https://github.com/xtranophilist/gnome-shell-extension-sensors) gnome extension. It lets one monitor temperatures provided via `lm_sensors` as well as through `nvidia-smi` and `bumblebee`.

###### Things done
- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

bjornfor reviewed (approved):
Nit: please add your maintainers entry in a separate commit (no need for separate PR, just split existing commit in two and put maintainers commit first).

justinas committed:
maintainers: add justinas

justinas committed:
gnomeExtensions.freon: init at 40

justinas commented:
> Nit: please add your maintainers entry in a separate commit (no need for separate PR, just split existing commit in two and put maintainers commit first).

Thanks for reviewing @bjornfor ! I have split the commit into two.

bjornfor commented:
@justinas: Thanks!
